specs/payment-api.yaml

```diff
@@ -98,6 +98,9 @@ paths:
         - requestor
         - provider
       summary: Get Payments for Debit Note.
+      deprecated: true
```

Comment:
Is it on purpose? It is Deprecated and unimplemented at the same time?

Reply:
Yep.